File: CTProyecto/app/src/main/java/pe/edu/upeu/ctproyecto/data/model/Evento.kt
package pe.edu.upeu.ctproyecto.data.model

import com.google.gson.annotations.SerializedName

data class Evento(
    @SerializedName("id") val id: Int,
    val titulo: String,
    val descripcion: String? = null,
    val fecha_inicio: String,
    val fecha_fin: String,
    val lugar: String? = null,
    val estado: String
)

File: WebTurismo/app/Models/Evento.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class Evento extends Model
{
    use HasFactory;

    protected $table = 'eventos';
    protected $primaryKey = 'id';
    public $timestamps = false;

    protected $fillable = [
        'titulo',
        'descripcion',
        'fecha_inicio',
        'fecha_fin',
        'lugar',
        'estado',
        'fecha_creacion'
    ];
}
